sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -1365,7 +1398,7 @@ Mono<Response<BinaryData>> getImageGenerationsWithResponse(String deploymentOrMo
      * completion of {@link Mono}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    Mono<Response<ImageGenerations>> getImageGenerationsWithResponse(String deploymentOrModelName,
+    public Mono<Response<ImageGenerations>> getImageGenerationsWithResponse(String deploymentOrModelName,
```

Comment:
This is approved for beta release. We need to discuss if we want a `WithResponse` overload that takes `RequestOptions` for methods with models before GA.